library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -523,4 +536,32 @@ public static void appendLanguageEntries(@NotNull Map<String, String> map) {
 			}
 		}
 	}
+
+	/* Static pack stuff */
+	private static List<ResourcePack> findUserStaticPacks() {
+		List<ResourcePack> returnList = new ArrayList<>();
+		try (Stream<Path> walkStream = Files.walk(QuiltLoader.getGameDir().resolve(STATIC_PACK_ROOT), 1)) {
+			for (Path path : walkStream.toList()) {
+				File pathAsFile = path.toFile();
+				String n = calcUserspacePackName(pathAsFile);
+				if (pathAsFile.isFile()) {
+					if (pathAsFile.toPath().toString().endsWith(".zip")) {
+						returnList.add(new ZipResourcePack(n, pathAsFile, false));
+					} else if (!pathAsFile.getName().equals(".DS_Store")) {
```

Comment:
if we're going to silently ignore filesystem helper files "like .DS_Store" instead of ignoring specifically .DS_Store, can we do something like
```java
private static final Set<String> IGNORED_FILES = Set.of(".DS_Store");

...

} else if (!IGNORED_FILES.contains(pathAsFile.getName())) {
  ...
}
```

Otherwise, if there will definitely never be any other fs helper files, we could just delete the comment

Reply:
are thumbs.db files still around?

Reply:
What does vanilla do for detecting packs in the resource packs folder? I feel like we should probably copy that

Reply:
Vanilla doesn't actually ignore .DS_Store files and similar (it throws an error if they're inside packs!), but it silently doesn't load loose files as a pack, same as is done here. Telling users that loose files aren't supported is a thing since other static data implementations do allow for them, so it felt pertinent to say 'hey this API doesn't do that'.

Reply:
It's worth noting that both the macOS and Windows filesystems are case-insensitive by default, so a plain equals won't suffice, either. And... there are many more files like this, including but not limited to `._.*` on macos, `.directory` on some Linux DEs, `desktop.ini` on windows, etc etc

Reply:
Anyways, the fs helper file thing is implemented with `thumbs.db` and `.DS_Store` in the set, and has a message to make an issue on the main github if an OS filesystem helper is warned of as a loose file

Reply:
> It's worth noting that both the macOS and Windows filesystems are case-insensitive by default, so a plain equals won't suffice, either. And... there are many more files like this, including but not limited to `._*` on macos, `.directory` on some Linux DEs, `desktop.ini` on windows, etc etc

Yaaay... eating some food rn but will go back and do another pass on this afterwards—thankfully `Set` has a case-insensitive contains method built-in, so that part won't be too big of an issue...